man/generateRWrappers.Rd

```diff
@@ -0,0 +1,120 @@
+% Generated by roxygen2: do not edit by hand
+% Please edit documentation in R/PythonPkgWrapperUtils.R
+\name{generateRWrappers}
+\alias{generateRWrappers}
+\title{Generate R wrappers for Python classes and functions}
+\usage{
+generateRWrappers(pyPkg, module, setGenericCallback, modifyFunctions = NULL,
+  modifyClasses = NULL, functionPrefix = NULL, pySingletonName = NULL,
+  transformReturnObject = NULL)
+}
+\arguments{
+\item{pyPkg}{The Python package name}
+
+\item{module}{The name of the Python module to be wrapped.}
+
+\item{setGenericCallback}{The callback to setGeneric defined in the target R package}
+
+\item{modifyFunctions}{Optional function to modify the returned functions}
+
+\item{modifyClasses}{Optional function to modify the returned classes}
+
+\item{functionPrefix}{Optional text to add to the name of the functions}
+
+\item{pySingletonName}{Optional singleton object in python}
```

Comment:
If the 'module' parameter is a class then this must be the name of a Python variable referencing an instance of the class.  If  'module' parameter is not a class then this must be NULL.